### PR TITLE
Variation Documentation

### PR DESCRIPTION
**Pull request by aparton**

### CODE REVIEW

**Comment by ens-ma7** [comment]
Hi @aparton does this need to go for release 103?

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 1]
```diff
@@ -209,7 +209,7 @@ <h3 id="cache_content">Data in the cache</h3>
       </tr>
       <tr>
         <td><b>ClinVar</b></td>
-        <td>2020-06</td>
+        <td>2020-08</td>
```
Comment (ima23) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:212]: For this release only it should be '2020-08/2020-09'.

File: ensembl/htdocs/ssi/species/Ursus_americanus_assembly.html [hunk 2]
```diff
@@ -1 +0,0 @@
-<p>The ASM334442v1 assembly was submitted by Jackson Laboratory on July 2018. The assembly is on scaffold level, consisting of 231,673 contigs assembled into 111,495 scaffolds. The N50 size is the length such that 50% of the assembled genome lies in blocks of the N50 size or longer. The N50 length for the contigs is 27,247 while the scaffold N50 is 189,900.</p>
```
Comment (ima23) [inline ensembl/htdocs/ssi/species/Ursus_americanus_assembly.html:1]: This is not variation docs related. Can it be it's removed by mistake?

**Review by ima23 — COMMENTED** [review]
@aparton : I had a look and overall all looks good for e103! A few general points:  
- Can you update the branch with latest code from master? It looks like 2 already merged PRs are not included. If they affect the pdfs, can they be refreshed? See variation synonyms and variant recorder PRs.
- There is no 'docs/htdocs/info/docs/api/variation/diagrams/Variation.dot' in the previous releases, did you want to update a png or something else?
- VEP plugins html updates are not in.

**Review by ima23 — APPROVED** [review]

**Pull request merged**